README.md

```diff
@@ -1,12 +1,12 @@
 # Grails Scim
 
-Grails Scim is a Grails 3.3.x plugin library for dealing with scim interface integration for user/group resources. It does expose rest endpoint which does consume by SCIM provider to send User/Group onboarding details on the server using SCIM contracts.
+Grails Scim is a Grails 6.2.0 plugin library for dealing with scim interface integration for user/group resources. It does expose rest endpoint which does consume by SCIM provider to send User/Group onboarding details on the server using SCIM contracts.
 
 ## Installation
 
 
 ```groovy
-compile 'org.grails.plugins:grails-scim:1.0-M3'
+implemenation 'org.grails.plugins:grails-scim:2.0.0'
```

Comment:
@RxL-Nikhil-Kanyal Give version as 2.0-M1

Reply:
@RxL-Nikhil-Kanyal Also add that for grails 3.x use compile 'org.grails.plugins:grails-scim:1.0-M3'

Reply:
done